hops/featurestore.py

```diff
@@ -1398,7 +1403,12 @@ def import_featuregroup_redshift(storage_connector, query, featuregroup, primary
     arguments = locals()
     arguments['type'] = "REDSHIFT"
     core._do_import_featuregroup(json.dumps(arguments))
-    job.launch_job(featuregroup)
+    #path to json file in hdfs
+    input_json_path = '--job_spec hdfs:///Projects/' + \
+                      os.environ[constants.ENV_VARIABLES.HOPSWORKS_PROJECT_NAME_ENV_VAR] + \
+                      '/Resources/featurestore-trainingdataset-job/configurations/' + \
```

Comment:
Should also be `'/Resources/featurestore-import/configurations/'` like above for the S3 case.

Reply:
Fixed thanks